doc/build-unix.md

```diff
@@ -72,6 +72,10 @@ Build requirements:
 
     sudo apt-get install build-essential libtool autotools-dev automake pkg-config libssl-dev libevent-dev bsdmainutils python3 libboost-system-dev libboost-filesystem-dev libboost-chrono-dev libboost-program-options-dev libboost-test-dev libboost-thread-dev
 
+Build requirements if you use the [depends system](/depends/README.md):
+
+    sudo apt-get install build-essential libtool autotools-dev automake pkg-config python3 bsdmainutils
```

Comment:
Wouldn't those instructions better be suited in the depends readme?

Reply:
The depends README currently shows which `apt` packages to install specifically for those depends. The items here are for the main application.